### ISSUE TITLE
Question about the SUN RGB-D metric of VA-DepthNet

### ISSUE THREAD

**Issue by jerry-ryu** [cnexah/VA-DepthNet#12]
I am writing to you today with a question regarding your impressive paper, "VA-DepthNet: A Variational Approach to Single Image Depth Prediction." I am particularly interested in **Table 4**, which presents **the results on the SUN-RGB-D test set when the model is trained only on the NYU V2 training dataset**.

While reading your paper, I noticed that the reported metrics for the SUN-RGB-D test set in Table 4 differ from those presented in other related works, such as the **AdaBins paper by Bhat et al. (2021)** and the **DDP paper by Yuanfeng Ji et al. (2023)**. I am wondering if there might be an explanation for these discrepancies or if I have perhaps missed some important detail in my understanding.

_Metrics written on VA-depthnet:_
![image](https://github.com/cnexah/VA-DepthNet/assets/62556539/4b4a8b3d-85fd-4e12-b932-b2c5577461d3)
_Metrics written on Adabins:_
![image](https://github.com/cnexah/VA-DepthNet/assets/62556539/a765163f-419f-496c-8a0a-72963266c088)
_Metrics written on DDP:_
![image](https://github.com/cnexah/VA-DepthNet/assets/62556539/267f8b68-91c6-40a9-996c-d9a9126cad5b)

I would be very grateful if you could clarify this point for me. I am eager to learn more about your work and its impressive methodology.

Thank you for your time and consideration.

**Comment by cnexah** [comment]
Sorry for the late reply!

Yes, the evaluation metrics on SUN-RBGD are different.
Please note we have mentioned in the paper, "In addition, we align the predictions from all the models with the ground truth by a scale and shift following Ranftl et al. (2020)."
